### PR TITLE
1-aedison: bug fix for switching less than three characters

### PR DESCRIPTION
**Pull request by adam-edison**
Hi, my first bug fix for a public repo here. Let me know if you need me to do anything more.

### CODE REVIEW

**Comment by knausj85** [comment]
I believe this fix might be better around here
https://github.com/knausj85/knausj_talon/blob/a31c7f162dd17c51a789fb7cfea43ad7621567b8/code/switcher.py#L182
to prevent the short words from entering the dictionary in the first place?

**Comment by adam-edison** [comment]
Agreed. I'll put the change there.

**Comment by knausj85** [comment]
you may need to restart talent after changing this file if you're not on the current beta; that's probably why the commands are being recognized

**Comment by adam-edison** [comment]
Even after restarting, the "focus" command is not working, and "running list" shows an empty list. And even after I switch back to my feature branch without this change, the same behavior occurs. Scratching my head...

**Comment by knausj85** [comment]
does that include apps that are launched after talon?

**Comment by adam-edison** [comment]
Just by performing that test, the problem disappeared. I think it's necessary to exit Talon Beta before switching branches.

**Comment by adam-edison** [comment]
The fix in the place indicated does not seem to filter the "i" application...

image

File: code/switcher.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def update_lists():
 
         words = get_words(name)
         for word in words:
-            if word and word not in running:
+            if word and word not in running and len(name) > 3:
```
Comment (knausj85) [inline code/switcher.py:183]: you need to check word, rather than name

Comment (knausj85) [reply code/switcher.py:183]: should probably maintain >= too

Comment (adam-edison) [reply code/switcher.py:183]: Wow, I feel embarrassed to have moved that fix so incorrectly. Sorry about that!

Comment (adam-edison) [reply code/switcher.py:183]: I would have tracked that down in my troubleshooting if I had separated out the "no commands are working" symptom from my previous efforts. Lessons learned.

**Comment by adam-edison** [comment]
Working now.

image

**Comment by adam-edison** [comment]
Should I squash this branch for you?

File: code/switcher.py (hunk 1 above)
Comment (knausj85) [reply code/switcher.py]: no worries, this happens to everyone. thanks for taking a look at this

**Pull request merged**